src/Client/Messages/PushedAuthorizationRequest.cs

```diff
@@ -0,0 +1,85 @@
+namespace IdentityModel.Client;
+
+/// <summary>
+/// Request for PAR
+/// </summary>
+/// <seealso cref="ProtocolRequest" />
+public class PushedAuthorizationRequest : ProtocolRequest
```

Comment:
It's convenient to have this type, because we can use ProtocolRequest.Prepare, and I am like that you can also have a nice object model if you want to set a commonly used parameter. However, I'm not sure what should happen if the parameters collection contains a parameter that is also set on one of the named properties.